oada-core/http-handler/oada-srvc-http-handler/package.json

```diff
@@ -44,15 +45,23 @@
     "oada-error": "^1.1.1",
     "stream": "^0.0.2",
     "timed-cache": "^1.1.0",
+    "ts-node": "^8.8.1",
     "type-is": "^1.6.18",
+    "typescript-json-schema": "^0.42.0",
     "uuid": "^3.0.1",
     "well-known-json": "^0.2.0"
   },
   "devDependencies": {
+    "@types/debug": "^4.1.5",
+    "@types/jsonpointer": "^4.0.0",
+    "@types/node": "^12.12.31",
+    "@types/ws": "^7.2.3",
     "chai": "^3.5.0",
     "mocha": "^7.0.1",
     "mocha-steps": "^1.0.2",
     "mock-require": "^2.0.2",
-    "supertest": "^3.0.0"
+    "prettier-standard": "^16.2.1",
```

Comment:
That or run `standard` and/or `prettier-standard` through `eslint`

I should share the monorepo setup I had created for ECEIL apps and some of my other private projects. It also most of this stuff to be factored out above the individual projects so things like styling rules just auto apply to everything and do not need to be duplicated. I will try to remember to make an issue for us to discuss soon.

Reply:
Sure if you know how to make it work with `eslint` do that.

Reply:
I'll happily change it to the more sane just `prettier`, otherwise, there are many great tutorials on running `prettier` and `standard` through `eslint`. As is, PRs/commits are going to have a ton of style changes all the time because only your editor will run `prettier-standard`

Reply:
It only runs it if prettier-standard is a dependency of that project